src/lib.rs

```diff
@@ -161,12 +151,12 @@ fn clockpoly(
     let nv = num_vertices_arc;
     // Number of vertices in the circle
     let nc = num_vertices_arc * num_segments;
-    let fi = seg * nv;
-    let ti = 1 + (seg + 1) * nv;
-    let seq = fi..ti;
+    let from_iterator = seg * nv;
+    let to_iterator = 1 + (seg + 1) * nv;
+    let seq = from_iterator..to_iterator;
     // Angle offset so first segment is North
     let o = std::f64::consts::PI / (num_segments as f64);
-    let seq_reverse = (fi..ti).rev();
+    let seq_reverse = (from_iterator..to_iterator).rev();
     for i in seq {
```

Comment:
There's lots of intermediate variables defined here. Personally, I would find it a bit easier to read if some of the ones used only once are just inlined in that single place. So here, why not `for i in from_iterator..to_iterator`?

Reply:
Agreed. This is now more concise, good style finding a balance between readability and conciseness, or moving in that direction:

https://github.com/zonebuilders/zonebuilder-rust/pull/29/commits/734dff9b743b5115b17b5b1ee2a50defab68e869